src/main.cpp

```diff
@@ -494,10 +494,13 @@ void runAgent () {
                 pc.moveAirTravel(censusData.unit_mf, air, censusData.demo);
             }
 
-            // Typical day
-            pc.morningCommute(mask_behavior);
-            pc.interactDay(mask_behavior);
-            pc.eveningCommute(mask_behavior);
+            // Process daytime / nighttime interactions.
+            // Only do daytime interactions on weekdays
+            if (ExaEpi::Utils::isWorkday(i)) {
+                pc.morningCommute(mask_behavior);
+                pc.interactDay(mask_behavior);
+                pc.eveningCommute(mask_behavior);
+            }
```

Comment:
Do you want to make this change in `AgentContainer::interactDay()` instead (put the `isWorkDay` condition around the lines calling work, school, and work neighborhood interactions? We can then later add social interactions for the `!workDay` case.